api/app/googleflood.py

```diff
@@ -55,3 +61,45 @@ def get_google_floods_gauges(
     return response
 
 
+from pydantic import BaseModel
+from typing import List
+
+class InundationMap(BaseModel):
+    level: str
+    serializedPolygonId: str
+
+class InundationMapSet(BaseModel):
+    inundationMaps: List[InundationMap]
+
+
+def get_google_floods_inundations(
+    inundationMapSet: InundationMapSet,
+) -> gpd.GeoDataFrame:
```

Comment:
Follow up from conversation, we're going to fetch the gauge by region here then fetch the maps — all backend